### PR TITLE
Add tide to kubernetes/kube-deploy

### PR DESCRIPTION
**Pull request by spiffxp**
Also add blunderbuss per request

Fixes: https://github.com/kubernetes/kube-deploy/issues/480

### CODE REVIEW

**Comment by spiffxp** [comment]
/cc @roberthbailey 
I leave it to you to `/hold cancel`

**Comment by spiffxp** [comment]
/assign @roberthbailey 
(whoops, meant to assign)

File: prow/plugins.yaml [hunk 1]
```diff
@@ -153,6 +154,10 @@ plugins:
   - trigger
   - approve
 
+  kubernetes/kubectl:
```
Comment (roberthbailey) [inline prow/plugins.yaml]: should this be kubernetes/kube-deploy?

Comment (spiffxp) [reply prow/plugins.yaml]: d'oh, yes

**Pull request merged**